functions/src/project/delete-project.ts

```diff
@@ -67,3 +70,4 @@ export const onDeleteProject: CloudFunction<DocumentSnapshot> = firestore
     deleteProjectPings(project);
     await deleteProjectRepositories(context.params.projectUid, project);
   });
+  
```

Comment:
GitHub needs a blank line ending, I will update this

Reply:
Strange why it shows incorrect here, but in the actual file it is fine 😕 - no problem, marking as resolved